### PR TITLE
Re-organize examples to their use-case

### PR DESCRIPTION
**Pull request by danielkoh-88**
Organize examples to their respective use cases so that they are more browse-able. Examples referring to APIs in the deprecation path are also moved accordingly

### CODE REVIEW

File: session/generate_jwt_hmac.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from hashlib import sha256
+
+load_dotenv()
+
+HOST_NAME = os.getenv("BX_API_HOSTNAME")
+PUBLIC_KEY = os.getenv("BX_PUBLIC_KEY")
+SECRET_KEY = bytes(os.getenv("BX_SECRET_KEY"), 'utf-8')
+
+session = requests.Session()
+nonce = str(1234567890)
```
Comment (dexterhuang-blockone) [inline session/generate_jwt_hmac.py]: Could u make this fetch from a datetime object as well so it's dyamic?

Comment (dexterhuang-blockone) [reply session/generate_jwt_hmac.py]: Same for the ecdsa example if it's there

File: orders/cancel_order_hmac.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "Content-type": "application/json",
+    "Authorization": f"Bearer {JWT_TOKEN}",
+    "BX-SIGNATURE": signature,
+    "BX-TIMESTAMP": timestamp,
+    "BX-NONCE": next_nonce,
+}
+
+response = session.delete(
+    HOST_NAME
+    + "/trading-api/v1/orders"
```
Comment (dexterhuang-blockone) [inline orders/cancel_order_hmac.py]: Deprecated endpoint as well.

File: custody/transfer_asset_hmac.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+next_nonce = str(nonce + 1)
+timestamp = str(int(datetime.now(timezone.utc).timestamp() * 1000))
+path = "/trading-api/v1/command"
+
+body = {
+    "timestamp": timestamp,
+    "nonce": next_nonce,
+    "authorizer": AUTHORIZER,
+    "command": {
+        "commandType": "V1TransferAsset",
```
Comment (dexterhuang-blockone) [inline custody/transfer_asset_hmac.py]: This is not custody action as funds are still within the exchange.

File: amm/terminate_amm_instruction_hmac.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    "Content-type": "application/json",
+    "Authorization": f"Bearer {JWT_TOKEN}",
+    "BX-SIGNATURE": signature,
+    "BX-TIMESTAMP": timestamp,
+    "BX-NONCE": next_nonce,
+}
+
+response = session.delete(
+    HOST_NAME
+    + "/trading-api/v1/amm-instructions"
```
Comment (dexterhuang-blockone) [inline amm/terminate_amm_instruction_hmac.py]: Deprecated too.

File: amm/get_amm_instruction.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    'tradingAccountId': TRADING_ACCOUNT_ID
+}
+
+headers = {
+    "Content-type": "application/json",
+    "Authorization": f"Bearer {JWT_TOKEN}",
+}
+
+response = session.get(
+    HOST_NAME + "/trading-api/v1/amm-instructions", params=params, headers=headers
```
Comment (dexterhuang-blockone) [inline amm/get_amm_instruction.py]: Deprecated too.

File: amm/create_amm_instruction_hmac.py [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+headers = {
+    "Content-type": "application/json",
+    "Authorization": f"Bearer {JWT_TOKEN}",
+    "BX-SIGNATURE": signature,
+    "BX-TIMESTAMP": timestamp,
+    "BX-NONCE": next_nonce,
+}
+
+response = session.post(
+    HOST_NAME + "/trading-api/v1/amm-instructions", json=body, headers=headers
```
Comment (dexterhuang-blockone) [inline amm/create_amm_instruction_hmac.py]: Same here.

**Review by dexterhuang-blockone — COMMENTED** [review]
There's more deprecated stuff to move into the `_deprecation_candidate` folder. Can look at the updated api docs in a lower env to see the full list of deprecated/decommissioned items. Folder structure looks much cleaner now :)

File: README.md [hunk 7]
```diff
@@ -126,5 +126,5 @@ export BX_TRADING_ACCOUNT_ID=< your credential >
 Next steps
 -----
 Explore other functionality of the [Bullish API](https://api.exchange.bullish.com/docs/api/rest/#overview) via sample code, for example:
-- Web socket to [get L2 Order Book](l2_orderbook_web_socket.py) 
-- [Create AMM instructions](hmac/rest/create_amm_instruction_v2.py)
\ No newline at end of file
+- Web socket to [get L2 Order Book](websocket/l2_orderbook_web_socket.py)
```
Comment (dexterhuang-blockone) [inline README.md]: This should be changed as that endpoint is deprecated?

File: orders/create_order_hmac.py [hunk 8]
```diff
@@ -72,4 +49,4 @@
 response = session.post(
     HOST_NAME + "/trading-api/v1/orders", json=body, headers=headers
```
Comment (dexterhuang-blockone) [inline orders/create_order_hmac.py]: Deprecated endpoint

File: README.md (hunk 7 above)
Comment (danielkoh-88) [reply README.md]: Yes, changed it

File: orders/create_order_hmac.py (hunk 8 above)
Comment (danielkoh-88) [reply orders/create_order_hmac.py]: Yes, updated

File: transfers/transfer_asset_hmac.py [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+SECRET_KEY = bytes(os.getenv("BX_SECRET_KEY"), 'utf-8')
+JWT_TOKEN = os.getenv("BX_JWT")
+AUTHORIZER = os.getenv("BX_AUTHORIZER")
+
+session = requests.Session()
+response = session.get(HOST_NAME + "/trading-api/v1/nonce")
+nonce = json.loads(response.text)["lowerBound"]
+next_nonce = str(nonce + 1)
+timestamp = str(int(datetime.now(timezone.utc).timestamp() * 1000))
+path = "/trading-api/v1/command"
```
Comment (dexterhuang-blockone) [inline transfers/transfer_asset_hmac.py]: This is deprecated

File: pyvenv.cfg [hunk 10]
```diff
@@ -0,0 +1,5 @@
+home = /opt/homebrew/opt/python@3.12/bin
+include-system-site-packages = false
+version = 3.12.4
+executable = /opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/bin/python3.12
+command = /opt/homebrew/opt/python@3.12/bin/python3.12 -m venv /Users/daniel.koh/code/api-examples
```
Comment (B1Bernard) [inline pyvenv.cfg]: Not really sure, if this file is supposed to be pushed in.

I am guessing this is where user will need to input their own directory of where they want to start their virtual env?

**Review by B1Bernard — APPROVED** [review]
1 small comment. LGTM

**Pull request merged**